chart/templates/replicated-deployment.yaml

```diff
@@ -60,10 +66,20 @@ spec:
           mountPath: /etc/replicated/config.yaml
           readOnly: true
           subPath: config.yaml
+        {{- if .Values.privateCAsConfigmap }}
+        - mountPath: /certs
+          name: additional-certs
+        {{- end }}
         env:
         {{- with .Values.extraEnv }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
+        {{- if .Values.privateCAsConfigmap }}
+        - name: SSL_CERT_DIR
+          value: /certs
+        - name: SSL_CERT_CONFIGMAP
```

Comment:
this one is probably not necessary. it was only set in kots to make it available as a template function.

Reply:
updated